scripts/bfb-multi-install

```diff
@@ -0,0 +1,302 @@
+
+	MAC=$RSHIM_MAC_PREF$(printf "%02x\n" $MAC_SFX)
+	IP=${IPv4_PREF}.$(($FIRST_IP + $i))
+	rshim_dev=$(echo $line | cut -d '/' -f 3)
+	rshim_idx=${rshim_dev/rshim/}
+	pcie_dev=$(echo $line | cut -d ' ' -f 2)
+
+	ex rshim -d $pcie_dev -i $rshim_idx &
+	while [ ! -e "/dev/$rshim_dev/boot" ]
+	do
+		sleep 1
+	done
+
+	#echo ubuntu_PASSWORD=\''$1$HBP9plt5$nQCG6/HPovdPPTdBdnxgv/'\' > $bfcfg
+	echo ubuntu_PASSWORD=\' $has \' > $bfcfg
+	cat >> $bfcfg << EOF
+NET_RSHIM_MAC=$MAC
+
+bfb_modify_os()
+{
+	log ===================== bfb_modify_os =====================
+	sed -i -e "s@192.168.100.2/30@$IP/24@" $CLOUD_INIT_NET_CONFIG
+}
+
+EOF
+
+
+
+cat > /tmp/bf$((i+1)).log << EOF
+========================
+Installing BFB on BF #$((i+1))
+Device: $pcie_dev
+MAC: $MAC
+IP: $IP
+RSHIM: $rshim_dev
+User: ubuntu
+Password: $password
+LOG: /tmp/bf$((i+1)).log
+========================
+EOF
+
+	cat /tmp/bf$((i+1)).log
+
+	(
+	echo "bfb-install -b $bfb -c $bfcfg -r $rshim_dev" | tee -a /tmp/bf$((i+1)).log
+	bfb-install -b $bfb -c $bfcfg -r $rshim_dev >> /tmp/bf$((i+1)).log 2>&1
+	if [ $? -ne 0 ]; then
+		echo "ERROR: Installation of $((i+1)) failed. See /tmp/bf$((i+1)).log"
+	fi
+	/bin/rm -f $bfcfg
+	echo "BF #$((i+1)) Installation finished"
+	) &
+
+
+	let i=i+1
+done < $RSHIM_DEVS
+
+echo "Please wait..."
+
+wait
+
+/bin/rm -f $RSHIM_DEVS
+
+
+
+
+
+#__________________
+conf_ubuntu(){
+  echo "Setting Ubuntu Networking"
+  if ! command -v netplan > /dev/null 2>&1; then
+    apt-get install netplan.io -y
+  fi
+  # Verify if seetings not already set.
+  if (grep --include=\*.yaml -rnw /etc/netplan/20-tmfifo.yaml -e 'tmfifo_net0'); then
+      echo "Static IP already set, skipping to avoid brake current settings"
+  else
+    echo create new /etc/netplan/20-tmfifo.yaml file
+    touch /etc/netplan/20-tmfifo.yaml
+    cat  > /etc/netplan/20-tmfifo.yaml <<- EOF
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+
+EOF
+    y=" "
+
+    until [ $x -gt $var ]
+    do
+      cat  >> /etc/netplan/20-tmfifo.yaml <<- EOF
+
+    tmfifo_net$x:
+      dhcp4: no
+EOF
+      y="$y""tmfifo_net""$x, "
+      x=$((x+1))
+    done
+    y="[$y]"
+    cat <<- EOF >> /etc/netplan/20-tmfifo.yaml
+  bridges:
+    tm-br:
+            interfaces: $y
+            addresses: [192.168.100.1/24]
+            dhcp4: no
+EOF
+  fi
+  netplan generate && netplan apply
+  echo "Done settings for Ubuntu"
+}
+
+conf_centos7(){
+  # Host tmfifo_net interfaces configuration
+  i=0
+  until [ $x -gt $var ]
+  do
+	  if [ ! -e /etc/sysconfig/network-scripts/ifcfg-tmfifo_net$x ]; then
+		  cat <<- EOF > /etc/sysconfig/network-scripts/ifcfg-tmfifo_net$x 
+DEVICE=tmfifo_net$i
+BOOTPROTO=none
+ONBOOT=yes
+NM_CONTROLLED=no
+BRIDGE=br_tmfifo
+EOF
+	      
+    fi
+	  x=$((x+1))
+    i=$((i+1))
+  done
+  if [ ! -e /etc/sysconfig/network-scripts/ifcfg-br_tmfifo ]; then
+    cat > /etc/sysconfig/network-scripts/ifcfg-br_tmfifo << EOF
+DEVICE="br_tmfifo"
+BOOTPROTO="static"
+IPADDR="192.168.100.1"
```

Comment:
how do we know this is an OK subnet to use? What if your default route used 192.168.100.1 as a gateway?